docs/student.md

```diff
@@ -86,6 +86,46 @@ The organizing committee does not recommend that students start research and dev
 
   - According to the final project evaluation results, the remaining project bonus will be paid in proportion.
 
+## Student bonus instruction
+
+1. How much bonus amount will each student receive?
+
+Each project will be awarded RMB 12000, RMB 9000 and RMB 6000 (before tax), which based on the level of project difficulty.
+
+2. When will the bonus be paid?
+
+Students who pass the mid-term assessment will receive 50% of the project bonus first. According to the final project evaluation results, the remaining project bonus will be paid in proportion. If the student fails the final evaluation, the rest bonus will not be paid.
```

Comment:
@YifeiXia For the final project evaluation, I think there will be only two kind result: Pass or Fail.If pass, student will receive the remaining 50%. The `will be paid in proportion` may cause misunderstanding.

Reply:
Students who pass the mid-term assessment will receive 50% of the project bonus first, the remaining 50% bonus will be paid after the students pass the final project evaluation. Accordingly, the student will not receive the rest bonus if he/she fails the final evaluation. 

@wjmjimmie Does it seem more clear?

Reply:
Yes, I think so.